### ISSUE TITLE
params is not defined

### ISSUE THREAD

**Issue by mammalofski** [Shreyas3108/house-price-prediction#1]
the variable (or some deprecated part of sklearn) named 'params' has been used here.
which raises the error `name 'params' is not defined`

**Comment by ghost** [comment]
so what can i do while params is not defined???

**Comment by sduAraika** [comment]
Hi! I've also got such error, how can we fixed it ?? 
By the way this tutorial very helpful for beginners. Thanks a lot!

**Comment by sqlballs** [comment]
Hi all, I ran into this issue as well.  If you look at this example from skit-learn on Gradient Boosting regression, http://scikit-learn.org/stable/auto_examples/ensemble/plot_gradient_boosting_regression.html#sphx-glr-auto-examples-ensemble-plot-gradient-boosting-regression-py , there is an example of a Gradient Boosting model.  params is a variable defined by the parameters passed to the GradientBoostingRegressor.

If you make a variable as follows it works.
params = {'n_estimators': 400, 'max_depth':5, 'min_samples_split':2, 'learning_rate':0.1, 'loss':'ls'}

But you want that to match the values from In [86]

**Comment by Shreyas3108** [comment]
@sqlballs has solved it , Hence closing. Never ever dared to see the issues part.
